### ISSUE TITLE
[Programmatic access - Customer Insight - Segment Overlap]: Incorrect role is defined for the checkboxes present in "Segment Overlap" dialog.

### ISSUE THREAD

**Issue by mohit406** [microsoft/fluentui#18140]
### User Experience:
If the element does not contain required child roles then the user who depends upon Screen Reader for navigation might gets affected hence failed to get the proper information of the element available on the page.

Note: Login Credentials must not be provided within the bug.

### Repro Steps:

1. Navigate to URL "https://tip1.ci.ai.dynamics.com/app/intelligence?instanceId=944f558a-8395-4b82-9458-c2588e4d6507&IsDataPreparationEnabled_ProfileCustomer=true&Reactcustommodels=true" and login with valid credentials.
2. Select Intelligence -> Custom Models.
3. New screen starts appearing. Navigate to "New workflow" and select it. New dialog starts appearing. Navigate them.
4. Run AI tool and Verify the Issue.

### 
### Actual Result:
Elements with an ARIA role that require child roles are not contained by them for the "Workspaces" dropdown present in the "New workflow" dialog.

### Expected Result:
Ensures elements with an ARIA role that require child roles should be contained by them for the "Workspaces" dropdown  present in the "New workflow" dialog.

**Issue:** 
Ensures elements with an ARIA role that require child roles are contain by them (aria-required-children - 
https://accessibilityinsights.io/info-examples/web/aria-required-children)
Target application: 
Dynamics 365 Customer Insights - Custom models  - https://tip1.ci.ai.dynamics.com/app/intelligence?instanceId=245fac4d-5a09-4e5a-b2b6-626b663c1261&IsDataPreparationEnabled_ProfileCustomer=true&Reactcustommodels=true

**Element path:** 
#Dropdown5581-list

**Snippet:** 
<div aria-labelledby="Dropdown5581-label" id="Dropdown5581-list" class="ms-FocusZone css-154 ms-Dropdown-items dropdownItems-530" role="listbox" data-focuszone-id="FocusZone5584">

**How to fix:**
Fix any of the following:
Required ARIA child role not present: option
Environment: 
Chrome version 89.0.4389.90

This accessibility issue was found using Accessibility Insights for Web 2.24.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7d

### Reference Links:
Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
Please reach out to C&AI Teams channel for any process related queries.

**Comment by mohit406** [comment]
A11yMAS, Accessibility, HCL, E2E-HCL-Customerinsights-MAR2021,MAS4.1.2, A11ysev2, Win10-chrome

**Comment by ecraig12345** [comment]
Without an isolated repro, we can't tell whether this is an issue with Fluent UI itself or with the way the product is using the control. Please see if you can create an isolated repro similar to the way the product uses the control, starting from either https://aka.ms/fluentpen or "Export to CodePen" on one of the [examples on our website](https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown). (And in the future, please remember that **GitHub is a public forum** and don't post screenshots or links for products.)

Also, can you please link your github account to your Microsoft account on [this page](https://repos.opensource.microsoft.com/people) so we can get your full name to message you on Teams? There's a different filing process which is a better fit for this type of issue, especially if you don't already have an isolated repro.

**Comment by mohit406** [comment]
@ecraig12345 

We have repro the issue on Fluent UI homepage and observed that issue is repro on the website
Please follow the below step to repro the issue:
1.Open the URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown
2. Navigate to the Basic Dropdowns under Usage
3. Run the Accessibility Insight tool(https://accessibilityinsights.io/)
4. Verify the Elements with an ARIA role that require child roles are not contained by Combo box.

**Comment by ecraig12345** [comment]
This is a duplicate of #17838, #17981, and #18333, and it's a false positive from Accessibility Insights. See https://github.com/microsoft/fluentui/issues/17838#issuecomment-824442277 for details.
